file.go

```diff
@@ -1125,6 +1134,20 @@ func (f *File) FlattenBatches() (*File, error) {
 	return of, nil
 }
 
+//Validates that the batch numbers are ascending
+func (f *File) isSequenceAscending() error {
+
+	lastSeq := 0
+	for _, batch := range f.Batches {
+		current := batch.GetHeader().BatchNumber
+		if current <= lastSeq {
+			return NewErrFileBatchNumberAscending("BatchNumber", lastSeq, current)
+		}
+		lastSeq = current
+	}
+	return nil
```

Comment:
nit: some minor formatting changes 
```suggestion
	lastSeq := 0
	for _, batch := range f.Batches {
		current := batch.GetHeader().BatchNumber
		if current <= lastSeq {
			return NewErrFileBatchNumberAscending("BatchNumber", lastSeq, current)
		}
		
		lastSeq = current
	}
	
	return nil
```